backend/app/services/device_service.py

```
@@ -476,6 +505,46 @@ def get_device_by_device_id(
+
+        If no explicit default exists for the type and there is exactly one active
+        device of the requested type, that device is returned.
+        """
+        devices = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Device",
+                    Kind.namespace == "default",
+                    Kind.is_active == True,
```

Ruff (0.15.4)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)